M/Makemsix/build_tarballs.jl

```diff
@@ -0,0 +1,159 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Makemsix"
+version = v"1.7.241"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/microsoft/msix-packaging.git",
+              "efeb9dad695a200c2beaddcba54a52c8320bd135"),
+    DirectorySource(joinpath(@__DIR__, "patches"))
+]
+
+
+# Script that will adapt to each platform
+script = raw"""
+    cd $WORKSPACE/srcdir/msix-packaging
+
+    # Update C++ standard to 17
+    sed -i 's/set(CMAKE_CXX_STANDARD 14)/set(CMAKE_CXX_STANDARD 17)/' CMakeLists.txt
+    find . -name "CMakeLists.txt" -type f -exec sed -i 's/set(CMAKE_CXX_STANDARD 14)/set(CMAKE_CXX_STANDARD 17)/' {} \;
```

Comment:
Sounds like this should be fixed upstream, please submit a PR, we don't want to maintainer other people's code.

Reply:
It may be possible to depreciate those lines, as they were added due to external dependencies.